src/mdb_v8.c

```diff
@@ -3362,6 +3444,60 @@ jsobj_print_jsarray(uintptr_t addr, jsobj_print_t *jsop)
 }
 
 static int
+jsobj_print_jstyped_array(uintptr_t addr, jsobj_print_t *jsop)
```

Comment:
Typed arrays are used elsewhere, for instance to store [DomainFlags](https://github.com/nodejs/node/blob/master/src/node.cc#L940) and [tick info](https://github.com/nodejs/node/blob/master/src/node.cc#L969), but not for storing anything that is created by users, except for Buffers.

I would not display the length property for typed arrays with `::jsprint`, but maybe it would be useful to display the length as the output of `::nodebuffer` in case the buffer is a typed array? It seems handy to know how many bytes in memory to examine when looking at a buffer instance.

I would keep `jsobj_print_jstyped_array` printing other properties (e.g for use cases like `var buf = new Buffer('foo'); buf.some_property = true;`), but maybe that can be done by using `jsobj_print_jsobject` to print instances of JSTypedArray instance of keeping `jsobj_print_jstyped_array`.

What do you think?